torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -171,23 +172,62 @@ void RequestCallbackNoPython::processPythonRemoteCall(
   C10_THROW_ERROR(Error, "Python call not supported!");
 }
 
-void RequestCallbackNoPython::processRRefBackward(
-      RpcCommandBase& rpc,
-      const int64_t messageId,
-      const std::shared_ptr<FutureMessage>& responseFuture) const {
-  C10_THROW_ERROR(Error, "Python call not supported!");
-}
-
 void RequestCallbackNoPython::processScriptRemoteCall(
     ScriptRemoteCall& scriptRemoteCall,
     const std::function<void(void)>& postProcessing,
     std::vector<at::IValue>& stack,
     const c10::intrusive_ptr<OwnerRRef>& ownerRRef) const {
   TORCH_CHECK(
-      scriptRemoteCall.hasOp(), "ScriptRemoteCall needs to have an op!");
+    scriptRemoteCall.hasOp(), "ScriptRemoteCall needs to have an op!");
```

Comment:
is this indent intentional? I recall clang-format used to give me 4 spaces for line-continuation. Did that change in recent updates?

Reply:
Thanks for catching that, it was not intentional. The old previous version is correct, I didn't run clang-format on the file.